unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Mentions/MentionsUtils.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace DCL.Social.Chat.Mentions
+{
+    public static class MentionsUtils
+    {
+        private const string MENTION_PATTERN = @"\B@[a-zA-Z\d]{3,15}(#[a-zA-Z\d]{4})?";
+        private static readonly Regex MENTION_REGEX = new (MENTION_PATTERN);
+
+        public static bool IsAMention(string text)
+        {
+            var match = MENTION_REGEX.Match(text);
+            return match.Success;
+        }
+
+        public static bool IsUserMentionedInText(string userName, string text) =>
+            text.ToLower().Contains($"@{userName.ToLower()}");
```

Comment:
We can simplify this line by using `text.Contains($"@{userName}", StringComparison.InvariantCultureIgnoreCase);`

Reply:
done